packages/api/tests/endpoints/collections/list-collections.js

```diff
@@ -150,16 +154,18 @@ test.serial('default returns list of collections from query', async (t) => {
 });
 
 test.serial('returns list of collections with stats when requested', async (t) => {
-  const stub = sinon.stub(EsCollection.prototype, 'getStats').returns([t.context.testCollection]);
-
   const response = await request(app)
     .get('/collections?includeStats=true')
     .set('Accept', 'application/json')
     .set('Authorization', `Bearer ${jwtAuthToken}`)
     .expect(200);
 
```

Comment:
Please include a collection which has no granules, its stats has 0 record for all statuses.

Reply:
yes